include/ginkgo/core/base/mpi.hpp

```diff
@@ -58,6 +58,19 @@ namespace gko {
 namespace mpi {
 
 
+/**
+ * Return if GPU aware functionality is available
+ */
+static constexpr bool is_gpu_aware()
```

Comment:
This will probably depend on the specific executor

Reply:
Perhaps we can also postpone that to later when we do something cleverer with our cmake setup for that.